### ISSUE TITLE
gulp configuration issues jshint not configured as a task

### ISSUE THREAD

**Issue by cloidjgreen** [googlecodelabs/psk-es2015#11]
"Error: Task jshint is not configured as a task on gulp."  which it's not

both 'jshint' and 'gulp-jshint' seem to be missing from the gulp dependencies manifest.  There is no initial  ".jshintrc" file to start with.  When 'jshint' is removed from the gulp default task list gulp then blows up with. 

"Error: Task rename-index is not configured as a task on gulp. "

**Comment by robdodson** [comment]
ah yep these two tasks were removed in the most recent release of PSK. We need to update the codelab to reflect that.

**Comment by robdodson** [comment]
@nicolasgarnier are you able to do this?

**Comment by cloidjgreen** [comment]
I initially started out to accomplish the online task list with the ES6 integration.  After working through the various jshint issues I found that the gulp task that looks for ES6 code was configured to parse through all of bower_components, blowing up naturally, so I just commented that out.

Got authenticated with firebase and was able to add tasks but the task were not showing up in the browser.  

BTW I did not follow the configuration directly.  Instead of commenting out the three pages in the starter kit route and doing a one page task list, I added a 4th route

This is when I got to the point that I did not have the time to invest anymore.

The tasks were not showing up on my fourth page as expected.

Trying to figure out why I was not getting the three way binding on the browser of the tasks that were clearly in Firebase I debugged around and found that the firebase ref did not survive Chrome debugging.  It simple became undefined after I would halt application at a break point.  This may be the norm for http connections when javascript is stopped, I'm not sure.

At any rate I am used to being able to find variables and trace through $scope in Angular and I am not able to grock how web components shadow dom scoping works well enough to be able to be rapidly effected in Polymer.  

And I need to be effective.

So I am setting Polymer aside for a while till I can undertake a deeper study of Web Components.

 Am falling back to Angular with Material Design.

I have been through all of your online labs BTW.  They are really great.  

Congratulations!
